app/controllers/scanned_resources_controller.rb

```diff
@@ -13,6 +14,13 @@ def after_create_success(obj, change_set)
     handle_upload_selene_files(obj)
   end
 
+  def propagate_selene_visiblity_and_state
```

Comment:
Okay, I think the way this works is that it pre-fills the edit form for the new resource with state and visibility values. It would be nice to have a comment on this method saying that's why we have it.

Reply:
Added. And changed the method name to `set_selene_visiblity_and_state`